common/src/main/java/com/box/l10n/mojito/json/ObjectMapper.java

```diff
@@ -124,4 +146,10 @@ public static ObjectMapper withNoFailOnUnknownProperties() {
     objectMapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
     return objectMapper;
   }
+
+  public static ObjectMapper withSmileEnabled() {
+    ObjectMapper objectMapper = new ObjectMapper(new SmileFactory());
```

Comment:
following previous comment, if blackbird is only for Smile, should we just register the module here?

Reply:
I could do it here, registering it in the constructor matches the existing convention of registering modules in the mappers constructor.